deploy/operator.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: observatorium-operator
+  labels:
+    app.kubernetes.io/component: observatorium-operator
+    app.kubernetes.io/name: observatorium-operator
+    app.kubernetes.io/part-of: observatorium
+    control-plane: observatorium-operator
+spec:
+  selector:
+    matchLabels:
+      control-plane: observatorium-operator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: observatorium-operator
+    spec:
+      containers:
+      - name: observatorium-operator
+        image: quay.io/nmagnezi/observatorium-operator:latest
```

Comment:
Do we need to update this?

Reply:
Yes, we will once we get approval on the rest of the PR, and when the operator will be pushed to the final location.
It is needed now for QE.

Reply:
Done